### PR TITLE
PYRO-118 egg swapper

### PR DESCRIPTION
**Pull request by thewander02**
WE HAVE DONE IT

### CODE REVIEW

File: resources/scripts/components/elements/DropdownMenu.tsx [hunk 1]
```diff
@@ -119,14 +119,14 @@ const DropdownMenuRadioItem = React.forwardRef<
...
-            'relative flex cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm outline-none transition focus:duration-0 focus:bg-[#ffffff33] focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50',
+            'relative flex cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm outline-none transition focus:duration-0 focus:bg-[#ffffff10] focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50',
             className,
         )}
         {...props}
     >
         <span className='absolute left-2 flex h-3.5 w-3.5 items-center justify-center'>
             <DropdownMenuPrimitive.ItemIndicator>
-                <DotFilledIcon className='h-4 w-4 fill-current' />
+                <a className='text-neutral-500'>{`>`}</a>
```
Comment (ferothefox) [inline resources/scripts/components/elements/DropdownMenu.tsx:129]: Not sure about this being an anchor tag. Make it a div or something

File: vite.config.ts.timestamp-1717110465566-8a096ec7016e6.mjs
Comment (ferothefox) [inline vite.config.ts.timestamp-1717110465566-8a096ec7016e6.mjs:1]: Remove this from git

**Review by ferothefox — CHANGES_REQUESTED** [review]
LGTM. Just needs one more clean passover. Ellie will review in a sec

File: resources/scripts/components/elements/DropdownMenu.tsx (hunk 1 above)
Comment (thewander02) [reply resources/scripts/components/elements/DropdownMenu.tsx:129]: yea i missed this should prob be div

**Review by ferothefox — APPROVED** [review]

**Comment by ferothefox** [comment]
let bro merge

**Comment by thewander02** [comment]
W

**Pull request merged**